packages/main/src/TableRowBase.ts

```diff
@@ -117,3 +109,8 @@ abstract class TableRowBase extends UI5Element {
 }
 
 export default TableRowBase;
+
+const isInstanceOfTableHeaderRow = (obj: any) => "isHeaderRow" in obj && obj.isHeaderRow() as boolean;
```

Comment:
do we still need those instanceof magics?

my idea was setting an attribute in the onEnterDom e.g.
this.toggleAttribute("sap-table-row-base", true);

and to determine whether it is header-row or normal row
$0.hasAttribute("sap-table-row-base");